pkg/internal/mock/kms/mock_crypto_box.go

```diff
@@ -0,0 +1,44 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package kms
+
+// MockCryptoBox is a mock CryptoBox.
+type MockCryptoBox struct {
+	EasyValue     []byte
+	EasyOpenValue []byte
+	SealOpenValue []byte
+	EasyErr       error
+	EasyOpenErr   error
+	SealOpenErr   error
+}
+
+// Easy seals a message with a provided nonce.
+func (m *MockCryptoBox) Easy(payload, nonce, theirPub []byte, myKID string) ([]byte, error) {
```

Comment:
What does this terminology mean? "Easy"

Reply:
@DRK3 it's easy ;-)
https://github.com/hyperledger/aries-framework-go/blob/master/pkg/kms/localkms/crypto_box.go#L58